app/uk/gov/hmrc/transitmovementsrouter/connectors/PersistenceConnector.scala

```diff
@@ -77,8 +78,9 @@ class PersistenceConnectorImpl @Inject() (httpClientV2: HttpClientV2, appConfig:
         .withBody(source)
         .execute[Either[UpstreamErrorResponse, HttpResponse]]
         .map {
-          case Right(_) =>
-            Right(())
+          case Right(res) =>
+            val m = Json.toJson(res.body) \ "messageId"
```

Comment:
if we don't gett a message ID, it's our problem, so would be a 5xx (but that would be a job for the controller, not this, which is a response to our request.

Reply:
I wonder, however, if this should get a domain object/DTO and we should return that, rather than just pulling the message ID out here, especially if we need to return more information later (thinking PPNS)